commit 7eec675669fbdef376198d3b04349301de3270f4
Author: Jonathan Neuschäfer <j.neuschaefer@gmx.net>
Date:   Sat Sep 5 22:43:26 2020 +0200

    docs: ubifs-authentication: Add a top-level heading
    
    This prevents the chapter headings from showing up in the table of
    contents in filesystems/index.html.
    
    Note that I didn't pick "UBIFS Authentication" as the document title,
    because there is a chapter of the same name, and Sphinx complains about
    multiple headings with the same name:
    
      /.../Documentation/filesystems/ubifs-authentication.rst:207:
      WARNING: duplicate label filesystems/ubifs-authentication:ubifs
      authentication, other instance in
      /.../Documentation/filesystems/ubifs-authentication.rst
    
    Remove the :orphan: tag, as the document has been included into the
    toctree.
    
    Signed-off-by: Jonathan Neuschäfer <j.neuschaefer@gmx.net>
    Link: https://lore.kernel.org/r/20200905204326.1378339-3-j.neuschaefer@gmx.net
    Signed-off-by: Jonathan Corbet <corbet@lwn.net>

diff --git a/Documentation/filesystems/ubifs-authentication.rst b/Documentation/filesystems/ubifs-authentication.rst
index 1f39c8cea702..5210aed2afbc 100644
--- a/Documentation/filesystems/ubifs-authentication.rst
+++ b/Documentation/filesystems/ubifs-authentication.rst
@@ -1,11 +1,13 @@
 .. SPDX-License-Identifier: GPL-2.0
 
-:orphan:
-
 .. UBIFS Authentication
 .. sigma star gmbh
 .. 2018
 
+============================
+UBIFS Authentication Support
+============================
+
 Introduction
 ============